spec/rubocop/cop/lint/symbol_conversion_spec.rb

```diff
@@ -46,6 +46,14 @@
     expect_no_offenses(<<~RUBY)
       :"foo-bar"
     RUBY
+
+    expect_no_offenses(<<~RUBY)
+      :'foo-bar'
+    RUBY
+
+    expect_no_offenses(<<~RUBY)
+      :'foo-bar""'
+    RUBY
```

Comment:
That just my two cents. Can you split the tests for the sake of test independence? e.g.

```suggestion
  end

  it 'does not register an offense for a symbol that requires single quotes' do
    expect_no_offenses(<<~RUBY)
      :'foo-bar'
    RUBY
  end

  it 'does not register an offense for a symbol that requires single quotes, when it includes double quotes' do
    expect_no_offenses(<<~RUBY)
      :'foo-bar""'
    RUBY
```

Reply:
fixed
https://github.com/rubocop/rubocop/pull/11970/commits/a90df2d8dde0f32beafbc1baf6f7354dc069e98d